### DISCUSSION TITLE
Rate limit

### DISCUSSION THREAD

**Discussion by Legends-of-NGDB** [xtekky/gpt4free#2297] [Q&A]
I am a JavaScript dev so I have 2 questions. Is it available in js and what is the rate limit?

**Comment by Lamparter** [comment]
Hello @Legends-of-NGDB, this is a Python library and therefore _isn't_ available for use in JavaScript. However, it exposes an [API](https://github.com/xtekky/gpt4free/blob/main/docs/interference-api.md) which you can consume through JavaScript if you're interested in that. As for the rate limit, there isn't one exactly as it depends on the gpt response model the module is consuming.

###### If you found this useful, feel free to mark it as an answer!
